scriptworker/task.py

```diff
@@ -80,7 +80,7 @@ def get_run_id(claim_task):
 
 # get_action_name {{{1
 def get_action_name(task):
```

Comment:
Can you update the name of this too.

Reply:
Done; also fixed tests, added coverage, and added a 16.0.0 changeset.